requirements-dev.txt

```diff
@@ -5,8 +5,9 @@
 sphinx
 
 # packages for testing and coverage
+flask
 nose2
```

Comment:
I also left it in the [CONTRIBUTING.md](https://github.com/anidata/rasp/blob/master/CONTRIBUTING.md) 😬